cmd/create.go

```diff
@@ -136,12 +136,17 @@ func create(cmd *cobra.Command, args []string) error {
 		logrus.Fatal("invalid defaults: ", err)
 	}
 
-	logrus.Info("generating ssh keys")
-	privkey, _ := generatePrivateKey(2048)
-	pubkey, _ := generatePublicKey(&privkey.PublicKey)
-	corr.PrivateKey = string(encodePrivateKeyToPEM(privkey))
-	corr.PublicKey = string(pubkey)
-
+	if corr.Vars["corral_private_key"] == nil && corr.Vars["corral_public_key"] == nil {
```

Comment:
Can you add the following log messages to each respective branch:
```
logrus.Info("reusing generated ssh keys")
```
and
```
logrus.Info("generating ssh keys")
```

Reply:
Done